rfc/text/advanced/ap_48_uris.md

```diff
@@ -2,6 +2,62 @@
 
 WAMP pre-defines the following error URIs for the **Advanced Profile**. WAMP peers SHOULD only use the defined error messages.
 
+## Authentication
+
+No authentication method the *Client* offered is accepted.
+
+{align="left"}
+        wamp.error.no_auth_method
+
+The *Client* authenticated for a *Realm* (`realm|string`) that does not (or no longer) exists.
```

Comment:
Mmm... I don't like the phrase `authenticated` because technically Client was not authenticated, the Client only tried to authenticate :)

So mb rephrase it to:
```
The *Client* tries to authenticate for a *Realm* (`realm|string`) that does not (or no longer) exists.
```
or 
```
The *Client* tried to authenticate for a *Realm* (`realm|string`) that does not (or no longer) exists.
```

Well... Of course, my English is not so good as @ecorm's :)

Reply:
agreed. I'll change that.